resources/views/pages-legacy/admin.blade.php

```diff
@@ -19,7 +19,7 @@
     if ($forUser == null) {
         $message = "Unknown user: $altsForUser";
     } else {
-        $altsForUser = $forUser->User;
```

Comment:
This should only display both if they differ.

Reply:
Fixed in latest.